### ISSUE TITLE
Is ALT text part of the visible label for 2.5.3 Label in Name?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#602]
This question from someone looking for edge cases. In a scenario where a user turns off their images, the value of the ALT text is displayed on screen. It could therefore be considered a visible label.

With this in mind, should it be part of the consideration for 2.5.3?

We are already tackling situations where an image contains images of text. That clearly applies, given the wording on images of text in the SC; however, the above interpretation would mean the ALT should be considered visible text and every image whose alt text is overridden with aria-label would result in the same failure that would happen if aria-label is used to replace the visible label on an input.

**Comment by awkawk** [comment]
I'd say no. There is already a requirement for the alt to be equivalent to the content of the image, I don't think we want to handle this edge case in 2.5.3.

**Comment by bruce-usab** [comment]
Does the TITLE attribute value factor into accessible name?  Is TITLE even in HTML5?

**Comment by mraccess77** [comment]
@bruce-usab yes and yes https://www.w3.org/TR/html-aam-1.0/#input-type-text-input-type-password-input-type-search-input-type-tel-input-type-url-and-textarea-element

**Comment by bruce-usab** [comment]
Thanks Jonathan, that citation clearly answer my question.

Probably old to both of you, but others may find this helpful:
http://www.ssa.gov/accessibility/andi
Accessible Name & Description Inspector 

It was such a bother for SSA to calculate accessible name manually, that they created a free tool!

**Comment by awkawk** [comment]
Proposed response: Thank you for the comment. As the label is defined as being intended to be presented to all users the Working Group does not regard the alternative text for an image as part of the label, even if there is a way for any user to display this text (by turning off the display of images in some browsers).
